test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -127,6 +127,22 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
     });
   }
 
+  function setupAdditionalAuthButton() {
```

Comment:
Do this qualifies for an e2e test? At the end we are just calling the callback they sent us, nothing else happens in the widget (and we don't get a token back).